src/loops/nested_loops/hard/q6/TryAngleButBothWaysAKAPyramid.java

```diff
@@ -0,0 +1,38 @@
+package loops.nested_loops.hard.q6;
+
+/*
+Write a program to ask the use for n ("Enter number of lines: "), and print the following pattern (Pyramid) using nested loops based on their input:
+           1
+          212
+         32123
+        4321234
+       543212345
+         … 1 ...
+n …   ...  1 … …  n
+
+ */
+
+import global.exceptions.InvalidClauseException;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.MethodSource;
+
+import java.util.Scanner;
+public class TryAngleButBothWaysAKAPyramid
+{
+    public static void main( String[] args )
+    {
+        Scanner input = new Scanner( System.in );
```

Comment:
I would also add a check to make sure the user inputs a positive number, and it should print `Invalid input!` otherwise.